schemas/models/structured_model.json

```diff
@@ -2,34 +2,48 @@
   "$schema": "http://json-schema.org/draft-07/schema",
   "$id": "https://raw.githubusercontent.com/bettyblocks/json-schema/acceptance/schemas/models/structured_model.json",
   "title": "Question",
-  "description": "An object representing a single question in a questionnaire, including the prompt to be displayed to the user and any necessary data to properly evaluate their response.",
-  "type": "object",
+  "description": "An Json Schema to validate client json schema in regards to structured models",
   "additionalProperties": false,
+  "type": "object",
   "properties": {
     "score": {
-      "description": "The numerical score assigned to the question",
-      "type": "integer",
-      "default": 0
+      "type": "object",
+      "properties": {
+        "description": {"type": "string"},
+        "type": {"type": "string"},
+        "default": {"type": "integer"}
+}
+    },
+    "title": {
+      "type": "object",
+      "properties": {
+        "description": {"type": "string"},
+        "type": {"type": "string"},
+        "default": {"type": "string"}
+}
     },
     "answer": {
-      "description": "The written response provided by the user to the question",
-      "type": "string"
+      "type": "object",
+      "properties": {
+        "description": {"type": "string"},
+        "type": {"type": "string"},
+        "default": {"type": "integer"}
```

Comment:
Is this correct?

Reply:
I've mapped it to the structure defined in the documentation.